classes/PodsUI.php

```diff
@@ -3625,7 +3740,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( strlen( $value ) < 1 ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 95bfe1d757607e620a5ece97bd8de027</sub>